packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_inspector_platform.dart

```diff
@@ -117,11 +117,7 @@ abstract class GoogleMapsInspectorPlatform extends PlatformInterface {
   }
 
   /// If the platform supports getting information about heatmaps.
-  bool supportsGettingHeatmapInfo() {
-    throw UnimplementedError(
-      'supportsGettingHeatmapInfo() has not been implemented.',
-    );
-  }
+  bool supportsGettingHeatmapInfo() => false;
```

Comment:
This change is not ground overlay related, but as this method was not implemented for web platform, integration tests for web were failing because of this.
`supportsGettingHeatmapInfo() has not been implemented.`
We could have added implementation for this to web platform, but instead having this defaults to false, seems better solution.

Reply:
Sounds good to me, this is exactly what this "capability check" is supposed to do.

Reply:
Yes, this was a mistake; "supports"-style interface methods should default to false instead of throwing. Thanks for fixing it!